### PR TITLE
fix: add projectId to config type definitions

### PR DESCRIPTION
**Pull request by kjin**
Somehow I left out `projectId` from the `Config` object type definitions (probably because it doesn't have a default value)... this change adds it.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -125,6 +125,12 @@ export interface Config {
    */
   ignoreContextHeader?: boolean;
 
+  /**
+   * The ID of the Google Cloud Platform project with which traces should
+   * be associated.
+   */
+  projectId?: string;
```
Comment (jinwoo) [inline src/config.ts]: Qq. Shouldn't this be required? What happens if the project id is not given?

Comment (kjin) [reply src/config.ts]: It's not required. Project ID can also be either read from the metadata service, or the environment.

Comment (kjin) [reply src/config.ts]: On that note, let me add a comment to further elaborate on this.

Comment (jinwoo) [reply src/config.ts]: Thanks.

**Pull request merged**